lib/koans/01_equalities.ex

```diff
@@ -1,6 +1,13 @@
 defmodule Equalities do
   use Koans
 
+  @intro  """
```

Comment:
I like the message here but the last sentence seems off grammatically. To me it seems like it should read 'The path laid in front of you is but one of many.' Changes are:
`layed` -> `laid` (http://public.wsu.edu/~brians/errors/layed.html)
`infront` -> `in front`
`us` -> `is`